base/common/sbin/pki-upgrade

```diff
@@ -23,6 +23,6 @@
 # set NSS_DEFAULT_DB_TYPE.
 . /usr/share/pki/scripts/config
 
-${PYTHON_EXECUTABLE} -m pki.cli.upgrade "$@"
+${PYTHON_COMMAND} -I -m pki.cli.upgrade "$@"
```

Comment:
Why'd this one get changed to `${PYTHON_COMMAND}` whereas the rest of the files below kept it as `${PYTHON_EXECUTABLE}`...?

Reply:
Ah, typo. Let me fix that.

Reply:
Fixed. Good catch!